model/database/cosmosdb-metrics.yaml

```
@@ -0,0 +1,28 @@
+groups:
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
```

see https://github.com/open-telemetry/semantic-conventions/issues/1494#issuecomment-2422829241
```suggestion
    metric_name: db.client.cosmosdb.operation.request_charge
```